deploy/staging/deployment.yaml

```diff
@@ -0,0 +1,601 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: prison-visits-booking-staff
+  labels:
+    app: prison-visits-booking-staff
+  annotations:
+    kubernetes.io/change-cause: "<to be filled in deploy job command>"
+spec:
+  replicas: 1
+  revisionHistoryLimit: 1
+  minReadySeconds: 10
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      app: prison-visits-booking-staff
+  template:
+    metadata:
+      labels:
+        app: prison-visits-booking-staff
+    spec:
+      containers:
+        - name: prison-visits-booking-staff
+          image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/prison-visits-booking/prison-visits-staff:latest
+          imagePullPolicy: Always
+          command: ['sh', '-c', "bundle exec rails db:migrate && bundle exec rails db:seed && bundle exec puma -p 3000 -C ./config/puma_prod.rb --pidfile /tmp/server.pid"]
+          ports:
+            - containerPort: 3000
+#          livenessProbe:
```

Comment:
Reminder this probably needs to go back in at some point..

Reply:
Yep - will need to fix those 503s!